### PR TITLE
Move nlpaug to lazy import

### PR DESCRIPTION
**Pull request by sxjscience**
*Issue #, if available:*

https://github.com/autogluon/autogluon/issues/2706

*Description of changes:*

- [x] Move nlpaug-related code to lazy import.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Innixma** [comment]
Have we measured the impact on import speed with this change?

**Review by Innixma — APPROVED** [review]
LGTM!

**Comment by sxjscience** [comment]
In the process of testing on Colab. Will share results shortly.

File: multimodal/src/autogluon/multimodal/data/trivial_augmenter.py [hunk 1]
```diff
@@ -278,6 +276,10 @@ def augment_image(self, data):
         return op.augment(scale, data)
 
     def augment_text(self, data):
+        import nlpaug.augmenter.word as naw
+
+        from ..utils.nlpaug import InsertPunctuation
```
Comment (Innixma) [inline multimodal/src/autogluon/multimodal/data/trivial_augmenter.py:281]: minor optimization, can have these imported after

```python3
if op == "identity":
       return data
```

That way if we aren't using anything but identity, we don't need to import.

Comment (sxjscience) [reply multimodal/src/autogluon/multimodal/data/trivial_augmenter.py]: Sounds good. Fixed.

**Comment by sxjscience** [comment]
Verified that nlpaug is gone:

```
ncalls  tottime  percall  cumtime  percall filename:lineno(function)
        1    0.000    0.000   28.388   28.388 /content/autogluon/multimodal/src/autogluon/multimodal/__init__.py:1(<module>)
  168/102    0.000    0.000   21.047    0.206 /usr/lib/python3.8/importlib/__init__.py:109(import_module)
        1    0.000    0.000   11.468   11.468 /content/autogluon/multimodal/src/autogluon/multimodal/models/__init__.py:1(<module>)
        1    0.000    0.000   11.377   11.377 /usr/local/lib/python3.8/dist-packages/accelerate/__init__.py:5(<module>)
        1    0.000    0.000   11.361   11.361 /usr/local/lib/python3.8/dist-packages/accelerate/utils/__init__.py:5(<module>)
        1    0.000    0.000    9.846    9.846 /content/autogluon/multimodal/src/autogluon/multimodal/optimization/__init__.py:1(<module>)
        1    0.001    0.001    7.595    7.595 /usr/local/lib/python3.8/dist-packages/tensorflow/__init__.py:15(<module>)
        1    0.000    0.000    7.004    7.004 /content/autogluon/multimodal/src/autogluon/multimodal/data/__init__.py:1(<module>)
        1    0.000    0.000    5.819    5.819 /usr/local/lib/python3.8/dist-packages/torch/distributed/launcher/__init__.py:10(<module>)
        1    0.001    0.001    4.934    4.934 /usr/local/lib/python3.8/dist-packages/tensorflow/python/__init__.py:15(<module>)
        1    0.000    0.000    3.841    3.841 /usr/local/lib/python3.8/dist-packages/torch/distributed/elastic/multiprocessing/__init__.py:9(<module>)
        1    0.000    0.000    2.376    2.376 /usr/local/lib/python3.8/dist-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py:9(<module>)
        1    0.000    0.000    2.192    2.192 /content/autogluon/multimodal/src/autogluon/multimodal/utils/__init__.py:1(<module>)
        1    0.000    0.000    2.027    2.027 /usr/local/lib/python3.8/dist-packages/evaluate/__init__.py:20(<module>)
```

**Pull request merged**